src/qml/pages/wallet/DesktopWallets.qml

```diff
@@ -122,9 +122,8 @@ Page {
         width: parent.width
         height: parent.height
         currentIndex: navigationTabs.checkedButton.index
-        Item {
+        Activity {
```

Comment:
it builds, but doesn't run
```
./bitcoin-qt -signet
QSocketNotifier: Can only be used with threads started with QThread
QQmlApplicationEngine failed to load component
qrc:/qml/pages/main.qml:83:9: Type DesktopWallets unavailable
qrc:/qml/pages/wallet/DesktopWallets.qml:125:9: Activity is not a type
~InitExecutor : Stopping thread
~InitExecutor : Stopped thread
```

Reply:
Updated the PR to include the missing component